docs/optimization_guide/model_optimization_guide.md

```diff
@@ -11,34 +11,22 @@
    (Experimental) Protecting Model <pot_ranger_README>
 
 
-Model optimization is an optional offline step of improving final model performance by applying special optimization methods, such as quantization, pruning, preprocessing optimization, etc. OpenVINO provides several tools to optimize models at different steps of model development:
+Model optimization is an optional offline step of improving the final model performance and reducing the model size by applying special optimization methods, such as 8-bit quantization, pruning, etc. OpenVINO offers two optimization paths implemented in `Neural Network Compression Framework (NNCF) <https://github.com/openvinotoolkit/nncf>`__:
```

Comment:
Can we also add a note or whatever about the support of the optimized models coming from frameworks? Many times we got a question if the user needs to use our tools for in-framework optimized models.